src/Domain.LinnApps/Suppliers/Supplier.cs

```diff
@@ -76,5 +77,7 @@ public class Supplier
         public string Notes { get; set; }
 
         public int OrganisationId { get; set; }
+
+        public IEnumerable<SupplierContact> Contacts { get; set; }
```

Comment:
as mentioned on the React side - i'd call these supplierContacts whereever possible